resources/lib/codemirror/lib/codemirror.css

```diff
@@ -324,11 +324,6 @@ div.CodeMirror-dragcursors {
 .CodeMirror-line::selection, .CodeMirror-line > span::selection, .CodeMirror-line > span > span::selection { background: #d7d4f0; }
 .CodeMirror-line::-moz-selection, .CodeMirror-line > span::-moz-selection, .CodeMirror-line > span > span::-moz-selection { background: #d7d4f0; }
 
-.cm-searching {
```

Comment:
Could we override this CSS on the app side instead of altering the file here?

Reply:
I added it to the theme files in this commit - https://github.com/wikimedia/wikipedia-ios/pull/2858/commits/6e8f6e6be7e2fe677c3e2564befa4777fc98775b

it will get overridden here unless theme css fail to load, I can leave it here as fallback or move it to `codemirror-common.css` on our side

Reply:
It should stay here as the default fallback - the less we have to alter upstream Codemirror the better